vcr/types/interface.go

```diff
@@ -55,9 +52,11 @@ var ErrInvalidCredential = errors.New("invalid credential")
 // ErrInvalidPeriod is returned when the credential is not valid at the given time.
 var ErrInvalidPeriod = errors.New("credential not valid at given time")
 
-var vcDocumentType = "application/vc+json"
+// VcDocumentType holds the content type used in network documents which contain Verifiable Credentials
+var VcDocumentType = "application/vc+json"
```

Comment:
const

And I think this is duplicate?

Reply:
Yes, but we have a nasty import cycle if I use the one from `vcr/types/interface.go`.
Is something we should fix in later iterations.